tutorial/creating_packages/define_package_information.rst

```diff
@@ -1,2 +1,353 @@
+
+    add_executable(example src/example.cpp)
+    target_link_libraries(example hello::hello)
+
+You can see that we are linking with the target name ``hello::hello``. Conan sets this
+target name by default, but we can change it using the *properties model*. Let's try to
+change it to the name ``hellotarget``. To do this, we have to set the property
+``cmake_target_name`` in the package_info method of our *hello/1.0* Conan package:
+
+
+.. code-block:: python
+    :caption: *conanfile.py*
+    :emphasize-lines: 10
+
+    class helloRecipe(ConanFile):
+        ...
+
+        def package_info(self):
+            if self.options.shared:
+                self.cpp_info.libs = ["hello-shared"]
+            else:
+                self.cpp_info.libs = ["hello-static"]
+
+            self.cpp_info.set_property("cmake_target_name", "hellotarget")
+
+
+Then, change the target name we are using in the *CMakeLists.txt* in the *test_package*
+folder to ``hellotarget``:
+
+.. code-block:: cmake
+    :caption: test_package *CMakeLists.txt*
+    :emphasize-lines: 4
+
+    cmake_minimum_required(VERSION 3.15)
+    project(PackageTest CXX)
+    # ...
+    target_link_libraries(example hellotarget)
+
+And re-create the package:
+
+.. code-block:: bash
+    :emphasize-lines: 14
+
+    $ conan create . --build=missing
+    Exporting the recipe
+    hello/1.0: Exporting package recipe
+    hello/1.0: Using the exported files summary hash as the recipe revision: 44d78a68b16b25c5e6d7e8884b8f58b8 
+    hello/1.0: A new conanfile.py version was exported
+    hello/1.0: Folder: /Users/user/.conan2/p/a8cb81b31dc10d96/e
+    hello/1.0: Exported revision: 44d78a68b16b25c5e6d7e8884b8f58b8
+    ...
+    -------- Testing the package: Building --------
+    hello/1.0 (test package): Calling build()
+    ...
+    -- Detecting CXX compile features
+    -- Detecting CXX compile features - done
+    -- Conan: Target declared 'hellotarget'
+    ...
+    [100%] Linking CXX executable example
+    [100%] Built target example
+
+    -------- Testing the package: Running test() --------
+    hello/1.0 (test package): Running test()
+    hello/1.0 (test package): RUN: ./example
+    hello/1.0: Hello World Release! (with color!)
+
+You can see how Conan declares now the ``hellotarget`` instead of the default
+``hello::hello`` and the *test_package* builds successfully.
+
+The target name is not the only property you can set in the CMakeDeps generator. For a
+complete list of properties that affect the CMakeDeps generator behaviour, please check
+the :ref:`reference<CMakeDeps Properties>`. 
+
+Propagating environment or configuration information to consumers
+-----------------------------------------------------------------
+
+You can provide environment information to consumers in the ``package_info()``. To do so,
+you can use the ConanFile's :ref:`runenv_info<conan_conanfile_attributes_runenv_info>` and
+:ref:`buildenv_info<conan_conanfile_attributes_buildenv_info>` properties:
```

Comment:
Maybe add a note that it is not necessary to add bindirs to PATH or libdirs to LD_LIBRARY_PATH, those are automatically managed?